lib/livebook/runtime/erl_dist.ex

```diff
@@ -57,7 +57,23 @@ defmodule Livebook.Runtime.ErlDist do
   defp load_required_modules(node) do
     for module <- @required_modules do
       {_module, binary, filename} = :code.get_object_code(module)
-      {:module, _} = :rpc.call(node, :code, :load_binary, [module, filename, binary])
+
+      case :rpc.call(node, :code, :load_binary, [module, filename, binary]) do
+        {:module, _} ->
+          :ok
+
+        {:error, reason} ->
+          local_otp = :erlang.system_info(:otp_release)
+          remote_otp = :rpc.call(node, :erlang, :system_info, [:otp_release])
```

Comment:
wait, wait, wait. may rpc fails because it can't talk to node. So you should check if remote_otp is a charlist.

Reply:
If we got `{:error, error}` it means we can communicate, so it's unlikely that a subsequent call cannot communicate, right?

Reply:
Oh, good point. Ignore my changes then. I thought :rpc returned {:error, ...} for badrpc, but it is a separate tuple.

Reply:
Btw, relevant to our discussion: https://erlang.org/doc/man/erpc.html :)

Reply:
Neat, didn't know this module existed!